### ISSUE TITLE
CFR for Delete Icon

### ISSUE THREAD

**Issue by amedyne** [mozilla-mobile/focus-android#6209]
This is part of the Focus onboarding updates.
Display CFR(Contextual Feature Recommender)/Tool tip for the Delete Icon.

Figma design [here](https://www.figma.com/file/ahqwL3sByHprH2Rru9R58b/Onboarding-2022?node-id=2647%3A11565).

@jeffreygee please add comment or update once behavior on when the CFR appears is finalized.

**Comment by jeffreygee** [comment]
**Trigger logic**: CFR will pop up after user visits at least 3 sites. we should have it so that it's triggered by sessions on sites, and not opening the app 3 times.  
**Behavior to dismiss**: CFR can be dismissed by tapping a ‘close’ icon in the opposite corner of the pointer, or anywhere outside of the CFR. Once dismissed, will not be shown again.

**Comment by ionutbedregeanu** [comment]
@jeffreygee I exported  the cfr shape as SVG and I'm trying to import it in Android Studio but I'm receiving some errors
<img width="1064" alt="Screenshot 2022-02-02 at 16 44 25" src="https://user-images.githubusercontent.com/11731652/152176160-f52aeee8-33df-4ddd-a1ac-8b54d49e43b1.png">
Can you please take a look?

**Comment by ionutbedregeanu** [comment]
> **Trigger logic**: CFR will pop up after user visits at least 3 sites. we should have it so that it's triggered by sessions on sites, and not opening the app 3 times. **Behavior to dismiss**: CFR can be dismissed by tapping a ‘close’ icon in the opposite corner of the pointer, or anywhere outside of the CFR. Once dismissed, will not be shown again.

@jeffreygee Maybe we should consider to show this CFR when the user open 3 tabs and not 3 sites.
Can be scenarios when the user open a single site but 3 redirections are made, so 3 urls are were loaded and we are showing the CFR after one navigation.
What do you think?

**Comment by lobontiumira** [comment]
Verified on the 2/28 Focus Nightly build with several devices:
* Oppo Find X3 Lite (Android 11),
* Xiaomi Mi8 Lite (Android 10),
* Sony Xperia Z5 Premium (Android 7.1.1), and 
* Samsung Galaxy Tab A6 (Android 5.1.1).
The CFR for delete icon is displayed on the third opened tab.

I've encountered several bugs related to the CFR for delete icon:
* #6524 
* #6525 
* #6526

I'll close this ticket as resolved.
